src/components/app.website.component.html

```diff
@@ -1,6 +1 @@
-<mat-toolbar color="primary">
```

Comment:
Sure that we want to remove the toolbar? I think we just should make it fixed
Maybe just remove it for mobile

Reply:
I think, it looks way nicer like this on all devices. The App's name is in the Title for the browser